source/usage-examples/deleteMany.txt

```diff
@@ -0,0 +1,97 @@
+.. _csharp-delete-multiple:
```

Comment:
This page is following an older format. I worked with Alek last week to rewrite the UpdateMany page, which is now the format we're using for all Usage Example pages. It should be in upstream/master if you pull from there.